brouter-routing-app/src/main/aidl/btools/routingapp/IBRouterService.aidl

```diff
@@ -28,9 +31,9 @@ interface IBRouterService {
     //  "exportWaypoints" = 1 to export them (optional, default is no export)
     //  "pois"            = lon,lat,name|... (optional)
     //  "extraParams"     = Bundle key=value list for a profile setup (like "profile:")
-    //  "timode"          = turnInstructionMode [0=none, 1=auto-choose, 2=locus-style, 3=osmand-style, 4=comment-style, 5=gpsies-style, 6=orux-style] default 0
+    //  "timode"          = turnInstructionMode [0=none, 1=auto-choose, 2=locus-style, 3=osmand-style, 4=comment-style, 5=gpsies-style, 6=orux-style, 7=locus-new-style, 8=cruiser-stylem, 9=brouter-intern] default 0
     //  "heading"         = angle (optional to give a route a start direction)
-    //  "direction"       = (deprecated) angle
+    //  "direction"       = angle (optional, used on recalculation, only used by Locus)
```

Comment:
Would be good to indicate how it is used by Locus

Reply:
I have no view inside Locus. But I can see Locus sends a 'direction' tag via the interface.

Reply:
It is not ignored by brouter. 
It is stored at [BRouterWorker](https://github.com/abrensch/brouter/blob/eba0b18689d03d0ff3611a8896c4b593e45d8d15/brouter-routing-app/src/main/java/btools/routingapp/BRouterWorker.java#L67)
And used in this situation at [RoutingEngine](https://github.com/abrensch/brouter/blob/eba0b18689d03d0ff3611a8896c4b593e45d8d15/brouter-core/src/main/java/btools/router/RoutingEngine.java#L615) when an older calculation is found.

Reply:
Okay, I see it is used as start direction largely following "heading", how about making that clear in the description:

angle (optional, used like "heading" on a recalculation request by Locus as start direction)

Reply:
made an update in your way

Reply:
Thanks!